prow/github/report/report_test.go

```diff
@@ -171,12 +146,43 @@ func TestParseIssueComment(t *testing.T) {
 			expectedContexts: []string{"foo test"},
 			expectedUpdate:   123,
 		},
+		{
+			name:    "should create new comment for new commit",
```

Comment:
Can we have this scenario include something other than `bla test` (`foo test`?) I believe previously this would result in a `foo test` result at `sha123` and `bla test` result at `sha456`. Will be nice to show that we'll now only show a table with sha123 results.

Reply:
good catch - i've missed this case. now added as:
> should update existing comment to add failure about new job on the same commit

also slightly renamed the other similar case names.